### PR TITLE
Add evm account integration test

### PR DESCRIPTION
**Pull request by zjb0807**
Closes: #859 

https://github.com/open-web3-stack/open-runtime-module-library/blob/32666fd132119cfb0a6299b10909d903b0e9f5c8/tokens/src/lib.rs#L477

Tokens also add system provider.

### CODE REVIEW

File: runtime/mandala/src/lib.rs [hunk 1]
```diff
@@ -1146,9 +1147,9 @@ impl module_transaction_payment::Config for Runtime {
 pub struct EvmAccountsOnClaimHandler;
 impl Handler<AccountId> for EvmAccountsOnClaimHandler {
 	fn handle(who: &AccountId) -> DispatchResult {
-		if System::providers(who) == 0 {
-			// no provider. i.e. no native tokens
-			// ensure there are some native tokens, which will add provider
+		if Balances::total_balance(who).is_zero() {
```
Comment (xlc) [inline runtime/mandala/src/lib.rs:1150]: If user have other tokens (e.g. DOT), we don't need to do this

Comment (zjb0807) [reply runtime/mandala/src/lib.rs:1150]: Delete  the line `if Balances::total_balance(who).is_zero() {` 
 and all cases do `TransactionPayment::ensure_can_charge_fee` ?

Comment (xlc) [reply runtime/mandala/src/lib.rs:1150]: I mean the old code should be fine

Comment (zjb0807) [reply runtime/mandala/src/lib.rs:1150]: https://github.com/open-web3-stack/open-runtime-module-library/blob/32666fd132119cfb0a6299b10909d903b0e9f5c8/tokens/src/lib.rs#L477

If user have some tokens (e.g. ACA or DOT), `providers(who) = 1` and never 
 do `TransactionPayment::ensure_can_charge_fee`.

if user doesn't have any tokens, `providers(who) = 0` and do `TransactionPayment::ensure_can_charge_fee` doesn't make any sense.

**Review by xlc — APPROVED** [review]

**Pull request merged**